content/applications/marketing/marketing_automation/getting_started/testing_running.rst

```diff
@@ -1,41 +1,52 @@
-Testing and running a campaign
-==============================
+=========================
+Testing/running campaigns
+=========================
 
 Odoo gives users the ability to test marketing campaigns (and mailings) before officially running
 them.
 
-First, open the :guilabel:`Marketing Automation` application and click on a campaign. Make sure
-the campaign already has activities configured on it (or build a campaign by following the
-directions here on :doc:`this documentation <workflow_activities>`). To start a test, click the
-:guilabel:`Launch a Test` button at the top of the template form.
+First, open the :menuselection:`Marketing Automation` application, and select the desired campaign,
+which reveals that campaign's detail form.
+
+On the campaign detail form, make sure the campaign already has activities configured (or build a
+campaign by following the directions here on :doc:`this documentation
+</applications/marketing/marketing_automation/getting_started/workflow_activities>`).
+
+To start a test, click the :guilabel:`Launch a Test` button at the top of the campaign form.
 
 .. image:: testing_running/launch-test.png
    :align: center
    :alt: Launch a test button in Odoo Marketing Automation.
 
-When clicked, a pop-up window appears. In the dropdown field choose a specific record to run the
-test on, or create a brand new record by clicking the :guilabel:`Search More...` link at the bottom
-of the dropdown menu, and then click the :guilabel:`Create` button.
+When clicked, a :guilabel:`Launch a test` pop-up window appears. In the drop-down field, choose an
+existing record (or create a new one) to run the test on. To create a brand new record, click the
+:guilabel:`Search More...` link at the bottom of the drop-down menu, and then click the
+:guilabel:`Create` button.
+
+Doing so reveals a blank contact form, in which the :guilabel:`Name` and :guilabel:`Email` **must**
+be entered. When all the necessary information has been entered, click :guilabel:`Save`, and Odoo
+returns to the :guilabel:`Launch a test` pop-up, with that new record in the :guilabel:`Choose an
+existing record or create a new one to test` field.
 
-Once the record is selected, click :guilabel:`Continue`, and Odoo will redirect to the campaign
-test page.
+Once a record is selected, click :guilabel:`Continue`, and Odoo reveals the campaign test page.
 
 .. image:: testing_running/test-screen.png
    :align: center
    :alt: Test screen in Odoo Marketing Automation.
 
-Here, the name of the :guilabel:`Record` being tested is visible, along with the precise time this
-test workflow was started. Beneath that is the first activity (or activities) in the workflow.
+On the campaign test page, the name of the :guilabel:`Record` being tested is visible, along with
+the precise time this test workflow was started in the :guilabel:`Workflow Started On` field.
+Beneath that, in the :guilabel:`Workflow` section is the first activity (or activities) in the
```

Comment:
When combining singular and plural nouns, either:
- use a parenthesis on the s to indicate singular or plural, e.g. `choice(s)`, or
- write out the singular and plural if they don't follow this convention

```suggestion
Beneath that, in the :guilabel:`Workflow` section is the first activity or activities in the
```